pytorch_lightning/utilities/cloud_io.py

```diff
@@ -45,12 +46,8 @@ def load(
 
 
 def get_filesystem(path: Union[str, Path]) -> AbstractFileSystem:
```

Comment:
- update the typehint to _PATH from here: https://github.com/PyTorchLightning/pytorch-lightning/blob/ec1379da2c9388b144adcb4b4cc751839f085f65/pytorch_lightning/utilities/types.py#L37
as we use this across the codebase

- `url_to_fs` supports **kwargs being passed in. https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.core.url_to_fs

i think we should add **kwargs support to get_filesystem which are threaded through to url_to_fs as well

Reply:
Sounds good!